File: app/src/main/java/com/tt/handsomeman/ui/OnBoardingSlidePageFragment.java
package com.tt.handsomeman.ui;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.fragment.app.Fragment;

import com.tt.handsomeman.databinding.OnBoardingFragmentSlidePageBinding;

public class OnBoardingSlidePageFragment extends Fragment {

    private int image;
    private int description;
    private OnBoardingFragmentSlidePageBinding binding;

    static OnBoardingSlidePageFragment newInstance(int image, int description) {
        OnBoardingSlidePageFragment screenSlidePageFragment = new OnBoardingSlidePageFragment();
        Bundle args = new Bundle();
        args.putInt("image", image);
        args.putInt("description", description);
        screenSlidePageFragment.setArguments(args);
        return screenSlidePageFragment;
    }

    // Store instance variables based on arguments passed
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        image = getArguments().getInt("image", 0);
        description = getArguments().getInt("description");
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        binding = OnBoardingFragmentSlidePageBinding.inflate(inflater, container, false);

        ImageView imgOnBoarding = binding.imageViewOnBoarding;
        TextView txtOnBoarding = binding.textViewOnBoarding;
        imgOnBoarding.setImageResource(image);
        txtOnBoarding.setText(description);

        return binding.getRoot();
    }
}

File: app/src/main/java/com/tt/handsomeman/ui/OnBoardingSlidePageFragment.kt
package com.tt.handsomeman.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import android.widget.TextView

import androidx.fragment.app.Fragment

import com.tt.handsomeman.R

class OnBoardingSlidePageFragment : Fragment() {

    private var image: Int = 0
    private var description: Int = 0

    // Store instance variables based on arguments passed
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        image = arguments!!.getInt("image", 0)
        description = arguments!!.getInt("description")
    }

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val rootView = inflater.inflate(
                R.layout.on_boarding_fragment_slide_page, container, false) as ViewGroup

        val imgOnBoarding = rootView.findViewById<ImageView>(R.id.imageViewOnBoarding)
        val txtOnBoarding = rootView.findViewById<TextView>(R.id.textViewOnBoarding)
        imgOnBoarding.setImageResource(image)
        txtOnBoarding.setText(description)

        return rootView
    }

    companion object {

        internal fun newInstance(image: Int, description: Int): OnBoardingSlidePageFragment {
            val screenSlidePageFragment = OnBoardingSlidePageFragment()
            val args = Bundle()
            args.putInt("image", image)
            args.putInt("description", description)
            screenSlidePageFragment.arguments = args
            return screenSlidePageFragment
        }
    }
}
